app/views/hyrax/base/_form_progress.html.erb

```diff
@@ -31,13 +31,13 @@
           <%= check_box_tag 'agreement', 1, f.object.agreement_accepted, required: true %>
           <%= t('hyrax.active_consent_to_agreement') %><br>
           <%= link_to t('hyrax.deposit_agreement'),
-                      hyrax.agreement_path,
+                      '/distribution_license_request',
                       target: '_blank' %>
         </label>
       <% else %>
         <%= t('hyrax.passive_consent_to_agreement') %><br>
         <%= link_to t('hyrax.deposit_agreement'),
-                    hyrax.agreement_path,
+                    distribution_license_request_path,
```

Comment:
@flutterflies Instead of using the helper, should line 40 be `'/distribution_license_request',` to be consistent with the rest of your link_to's?